py3status/module.py

```diff
@@ -219,6 +252,9 @@ def run(self):
                         result['instance'] = self.module_inst
                         result['name'] = self.module_name
 
+                    # set universal module options in result
+                    result.update(self.module_options)
```

Comment:
Can you add this to the wiki or should I make a PR on the wiki (is this even possible)?